.Documentation/articles/getting-started/designer.md

```diff
@@ -121,9 +121,9 @@ The [Step Inspector](../innoactive-creator/step-inspector.md) allows you to set
 
 #### Save and Load
 
-The [Workflow Editor](../innoactive-creator/workflow-editor.md) and [Step Inspector](../innoactive-creator/step-inspector.md) automatically save all changes you make.
+The [Workflow window](../innoactive-creator/workflow-editor.md) and [Step Inspector](../innoactive-creator/step-inspector.md) automatically save all changes you make.
 
-You can open any selected training course by clicking the `Open in Workflow Editor` button in the `Inspector` tab of the `[TRAINING_CONFIGURATION]` game object. You can find this game object in your scene.
+You can open any selected training course by clicking the `Open in Workflow window` button in the `Inspector` tab of the `[TRAINING_CONFIGURATION]` game object. You can find this game object in your scene.
```

Comment:
```suggestion
You can open any selected training course by clicking the `Open in Workflow Window` button in the `Inspector` tab of the `[TRAINING_CONFIGURATION]` game object. You can find this game object in your scene.
```

Reply:
As this is a sentence, I would leave it minor.